installing/installing_azure_stack_hub/installing-azure-stack-hub-default.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-azure-stack-hub-default"]
+= Installing a cluster quickly on Azure Stack Hub
+include::modules/common-attributes.adoc[]
+:context: installing-azure-stack-hub-default
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Microsoft Azure Stack Hub that uses the default configuration options. However, you must manually configure the `install-config.yaml` file to specify values that are specific to Azure Stack Hub.
+
+[NOTE]
+====
+While you can select `azure` when using the installation program to deploy a cluster using installer-provisioned infrastructure, this option is only supported for the Azure Public Cloud.
+====
+
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[configured an Azure Stack Hub account] to host the cluster.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* You put the Cloud Credential Operator (COO) into xref:../../installing/installing_azure_stack_hub/manually-creating-iam-azure-stack-hub.adoc#manually-creating-iam-azure-stack-hub[manual mode]. Manual mode is used in environments where the cloud IAM APIs are not reachable.
```

Comment:
Uploading the RHCOS VHD image will require 16GB of local disk space to unzip (it can be deleted afterward). A file that size is also slow to upload. We should mention the disk space requirement. I would recommend that the installation be performed from a  VM on their environment to increase upload speed, but perhaps we don't need to be that prescriptive.

Reply:
Noted this under Prerequisites.